cmd/main.go

```diff
@@ -156,6 +162,21 @@ func preRun(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func resolveGlobExpression(excludedFileNames []string) map[string]bool {
+	var matchedFileNames = make(map[string]bool)
+	for _, filename := range excludedFileNames {
+		matched, err := filepath.Glob(filename)
+		if err != nil {
+			log.Error().Msgf("failed to get exclude path %s: %s", filename, err)
+		} else {
+			for _, value := range matched {
+				matchedFileNames[value] = true
+			}
+		}
+	}
+	return matchedFileNames
+}
```

Comment:
Please move it to a new file called `glob.go` or something under the `lib` folder.